en: fix deploy-tidb-binlog.md (#838)
pingcap/docs-tidb-operator: en/deploy-tidb-binlog.md

### Remove Pump nodes completely

> **Note:**
>

> > Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump nodes have been scaled in to `0` at this time, you need to scale out them at least to `1` before you remove them. To scale out the Pump to `1`, execute `kubectl edit tc ${tidb-cluster} -n ${namespace}` and modify the `spec.pump.replicas` to `1`.

Reviewer: ```suggestion
> - Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump cluster has been scaled in to `0` node at this time, you need to scale out the cluster at least to `1` node before you perform the removing operation in this section.
> - To scale out the Pump to `1`, execute `kubectl edit tc ${tidb-cluster} -n ${namespace}` and modify the `spec.pump.replicas` to `1`.
```

### Remove Pump nodes completely

> **Note:**
>

> > - Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump cluster has been scaled in to `0` node at this time, you need to scale out the cluster at least to `1` node before you perform the removing operation in this section.

Reviewer 2: I think "Pump cluster" is a bit strange, maybe we can just call it "Pump" and refer to the scale-in as "scale the Pump nodes to zero"?

> **Note:**
>

> > - Before performing the following steps, you need to have at least one Pump node in the cluster. If the Pump has scaled the Pump nodes to `0` node at this time, you need to scale out the Pump at least to `1` node before you perform the removing operation in this section.

Reviewer: ```suggestion
> - Before performing the following steps, you need to have at least one Pump node in the cluster. If you have scaled in Pump nodes to `0`, you need to scale out Pump at least to `1` node before you perform the removing operation in this section.
```
